commit f759fb91e055f1e6f7512fc89e3efde5f40a9074 (refs/changes/49/2949/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-29T23:28:39+00:00 (1 year, 10 months ago)
    
    joyent/node-fast#12 want support for node v6.12.0
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 2bb4b05..024fd84 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,9 +2,10 @@
 
 ## Not yet released.
 
-No changes.
+* #12 want support for node v6.12.0
 
 ## v2.3.1
+
 * #11 add RPC method name to metrics
 
 ## v2.3.0
diff --git a/package.json b/package.json
index 1fc6448..6065578 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
 		"jsprim": "^1.2.2",
 		"kang": "^1.1.0",
 		"lstream": "0.0.4",
-		"microtime": "2.0.0",
+		"microtime": "2.1.6",
 		"posix-getopt": "1.2.0",
 		"strsplit": "^1.0.0",
 		"vasync": "^1.6.3",
@@ -28,7 +28,7 @@
 		"restify": "5.2.0"
 	},
 	"optionalDependencies": {
-	    "dtrace-provider": "^0.6.0"
+	    "dtrace-provider": "^0.8.5"
 	},
 	"license": "MIT"
 }
